docs/source/training/socialisation/run_basic_inasafe.rst

```diff
@@ -99,7 +99,7 @@ the source of the data.
 |                              |               | the Australian Government.                                                      |
 |                              |               | `See more at... <http://inasafe.org/en/training/socialisation/datasets.html>`__ |
 +------------------------------+---------------+---------------------------------------------------------------------------------+
-| Java_population_WGS84        | WorldPop      | High resolution, modelled data for human population distributions.              |
+| Jakarta_population_WGS84        | WorldPop      | High resolution, modelled data for human population distributions.              |
```

Comment:
Found these error:

![image](https://cloud.githubusercontent.com/assets/2520159/15743464/d2a8375c-28ef-11e6-9de5-5199ae49ed0e.png)


just need little fix backspace like this:

![image](https://cloud.githubusercontent.com/assets/2520159/15743562/aeef79fa-28f0-11e6-85c6-b3475d48d162.png)